src/components/TabBar/Tab.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react'
+import styled, { css } from 'styled-components'
+
+import { InjectedProps, withTheme } from '../../hocs/withTheme'
+import { isTouchDevice } from '../../libs/ua'
+
+export interface Props {
+  id: string
+  label: string
+  selected?: boolean
+  disabled?: boolean
+  className?: string
+  onClick: (tabId: string) => void
+}
+
+type MergedProps = Props & InjectedProps
+
+const TabComponent: React.FC<MergedProps> = ({
+  id,
+  label,
+  onClick,
+  theme,
+  selected = false,
+  className = '',
+  ...props
+}) => {
+  const classNames = `${className} ${selected ? 'selected' : ''}`
+
+  const handleClick = () => {
+    onClick(id)
+  }
+
+  return (
+    <Wrapper
+      role="tab"
+      aria-selected={selected}
+      className={classNames}
+      onClick={handleClick}
+      theme={theme}
+      {...props}
+    >
+      {label}
+    </Wrapper>
+  )
+}
+
+export const Tab = withTheme(TabComponent)
```

Comment:
[IMO] The image associated with the name `Tab` is the whole including TabBar.
What about the name `TabItem` instead of `Tab`?